conjure-codegen/src/example_types/another/test_service.rs

```diff
@@ -1173,17 +1173,26 @@ where
         conjure_http::private::decode_empty_response(response_)
     }
 }
+use conjure_http::endpoint;
 ///A Markdown description of the service.
-pub trait TestService<I, O> {
+#[conjure_http::conjure_endpoints(name = "TestService")]
+pub trait TestService<#[request_body] I, #[response_writer] O> {
     ///The body type returned by the `get_raw_data` method.
     type GetRawDataBody: conjure_http::server::WriteBody<O> + 'static;
     ///The body type returned by the `get_aliased_raw_data` method.
     type GetAliasedRawDataBody: conjure_http::server::WriteBody<O> + 'static;
     ///The body type returned by the `maybe_get_raw_data` method.
     type MaybeGetRawDataBody: conjure_http::server::WriteBody<O> + 'static;
     ///Returns a mapping from file system id to backing file system configuration.
+    #[endpoint(
+        method = GET,
+        path = "/catalog/fileSystems",
+        name = "getFileSystems",
```

Comment:
Why is it necessary to include a camelCase `name` here?

Reply:
Yeah - it ensures that the name included in trace/request logs matches what's in the Conjure definition.